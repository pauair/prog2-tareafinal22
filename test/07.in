agregarNuevoProducto 50 prod50
definirFecha 14/05/2023
agregarUnidades 50 2 15/05/2023
agregarUnidades 50 5 15/05/2024
agregarUnidades 50 2 15/05/2025
agregarUnidades 50 5 15/05/2026
vencimientosProducto 50
vencimientosProducto 50
siguienteDia
fechaActual
vencimientosProducto 50
aumentarDias 370
fechaActual
vencimientosProducto 50
Fin