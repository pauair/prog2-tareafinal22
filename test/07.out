1>Producto agregado 50 - prod50.
2>Fecha definida en forma exitosa.
3>Al producto 50 se le agregaron 2 unidades con fecha de vencimiento 15/5/2023.
4>Al producto 50 se le agregaron 5 unidades con fecha de vencimiento 15/5/2024.
5>Al producto 50 se le agregaron 2 unidades con fecha de vencimiento 15/5/2025.
6>Al producto 50 se le agregaron 5 unidades con fecha de vencimiento 15/5/2026.
7>Información del producto 50 - prod50
Total de unidades: 14
15/5/2023 -> 2 
15/5/2024 -> 5 
15/5/2025 -> 2 
15/5/2026 -> 5 
8>Información del producto 50 - prod50
Total de unidades: 14
15/5/2023 -> 2 
15/5/2024 -> 5 
15/5/2025 -> 2 
15/5/2026 -> 5 
9>Se aumentó un día.
siguienteDia.
10>La fecha actual es 15/5/2023.
11>Información del producto 50 - prod50
Total de unidades: 12
15/5/2024 -> 5 
15/5/2025 -> 2 
15/5/2026 -> 5 
12>Se incrementó la fecha.
aumentarDias 370.
13>La fecha actual es 19/5/2024.
14>Información del producto 50 - prod50
Total de unidades: 7
15/5/2025 -> 2 
15/5/2026 -> 5 
15>Fin.
